server/src/test/java/org/elasticsearch/reservedstate/service/FileSettingsServiceTests.java

```diff
@@ -165,7 +165,6 @@ public void testInitialFileError() throws Exception {
     }
 
     @SuppressWarnings("unchecked")
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/95436")
```

Comment:
Was this supposed to be part of this PR?

Reply:
Oops, no, reverted.